### PR TITLE
[WIP] - Feature/users and groups UI updates

### PR DESCRIPTION
**Pull request by ShubhamThakre**
- User and Groups UI

### CODE REVIEW

**Comment by jjoyce0510** [comment]
@ShubhamThakre Do you mind posting screenshots here? Thanks!

File: datahub-web-react/src/app/entity/user/UserAssets.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+`;
+const ListItem = styled.div`
+    margin: 40px;
+`;
+
+type Props = {
+    searchResult: Array<SearchResult>;
+};
+export const UserAssets = ({ searchResult }: Props) => {
+    console.log('searchResult', searchResult);
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/user/UserAssets.tsx:39]: Get rid of this one!

File: datahub-web-react/src/app/entity/user/UserGroups.tsx [hunk 2]
```diff
@@ -1,80 +1,206 @@
...
+                                    ) : (
+                                        DummyGroupDescriptionText
+                                    )}
+                                </GroupDescription>
+                            </Row>
+                        </GroupItem>
+                    </GroupItemColumn>
+                    <GroupItemColumn xl={8} lg={8} md={8} sm={12} xs={24}>
+                        <GroupItem>
+                            <Row style={{ padding: '9px 11px 9px 11px' }}>
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/user/UserGroups.tsx:154]: We generally prefer to keep styling out of the JSX -- Use styled component above instead!

File: datahub-web-react/src/app/entity/user/UserGroups.tsx [hunk 3]
```diff
@@ -1,80 +1,206 @@
...
+                                                    )}
+                                                </GroupDescription>
+                                            </Row>
+                                        </GroupItem>
+                                    </Link>
+                                </GroupItemColumn>
+                            );
+                        })}
+                    {/* dummy */}
+                    <GroupItemColumn xl={8} lg={8} md={8} sm={12} xs={24}>
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/user/UserGroups.tsx:133]: This component is quite long - can we further break this out into small components?

Comment (gabe-lyons) [reply datahub-web-react/src/app/entity/user/UserGroups.tsx:133]: I think its only long because there's sample data in here that is copy + pasted :)

**Comment by ShubhamThakre** [comment]
Hi @jjoyce0510, 
Apologies for not sharing the screenshots and other code issues. 
The features I am working is still in progress, and @gabe-lyons ask me to raised the PR to follow my code progress. 
Thanks for your comments. I will keep those in mind. 
here is the short video for current updates, still its not completed and the work in progress.
https://user-images.githubusercontent.com/29713365/150907072-5a6f9d34-47c6-4b47-a0fe-7fe1cfa625e1.mp4

**Pull request closed without merging**